src/twisted/names/client.py

```diff
@@ -187,7 +190,7 @@ def parseConfig(self, resolvConf):
                 self.search = L.split()[1:]
                 self.domain = None
         if not servers:
-            servers.append(('127.0.0.1', dns.PORT))
+            servers.append(('::1', dns.PORT))
```

Comment:
It looks like you are changing things to append the IPv6 address for localhost instead of the IPv4 address for localhost.
This looks like a backwards incompatible change.